plugins/by-name/lint/deprecations.nix

```diff
@@ -0,0 +1,13 @@
+{ lib, ... }:
+let
+  inherit (lib) mkRemovedOptionModule;
+in
+{
+  imports = [
+    (mkRemovedOptionModule [
+      "plugins"
+      "lint"
+      "customLinters"
+    ] "plugins.lint.customLinters option was removed, use linters instead.")
```

Comment:
I haven't looked too close at the implementation, does this change require your upstream PR to be valid?

Reply:
Yes, it does. The current nvim-lint would consider any linter that uses an table assignment (e.g. `lint.linters.name = { prop = value }`), a new linter, even if it has a builtin one with the same name. That is probably why the `customLinters` option was needed.

Reply:
I'd recommend pulling that change out until we get an updated plugin, then.

Reply:
I thought we could wait for the plugin to be updated to merge this PR

Reply:
Just not sure how fast upstream is to respond to PRs and then there would be a wait time until it gets updated in nixpkgs and that package makes it to nixos-unstable. If you don't want to handle it in a separate PR, then probably just mark it a draft and come back to it later.

Reply:
I don't think it will take too long, but while it doesn't get merged I will convert the PR to a draft.

Reply:
Plugin author didn't want to merge because of breaking changes, so reverted to the old format.